lib/translator/index.js

```diff
@@ -63,6 +63,11 @@ function Translator(acceptLanguage, allTranslations, logger = console) {
  * @returns {Translator}
  */
 Translator.create = function (acceptLanguage, allTranslations, logger = console) {
+    //  Safari only sends the most preferred language's code, this is to have fallbacks in case we don't have that language
+    //  ex: fr-FR -> fr -> en-US -> en
+    if (acceptLanguage.length === 5) {
```

Comment:
This code seems extremely fragile, we need to find a better way to do this. Perhaps within Translator, after the header has already been parsed, we can more reliably do this kind of manipulation.

Reply:
Fixed to be less fragile and added unit test for this behavior.